unit_tests/cuda_test_helper.hpp

```diff
@@ -15,6 +15,10 @@
 
 #pragma once
 
+#ifndef __CUDACC__
+#error This is CUDA only header
+#endif
+
```

Comment:
restore new line at the file end please

Reply:
I think it is there, the deleted line was an additional new line.